release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -21,12 +21,12 @@ compliance, and governance requirements.
 == About this release
 
 Red Hat {product-title}
-(link:https://access.redhat.com/errata/RHBA-2020:2409[RHBA-2020:2409]) is now
+(link:https://access.redhat.com/errata/RHBA-2020:2909[RHBA-2020:2909]) is now
 available. This release uses link:https://v1-18.docs.kubernetes.io/docs/setup/release/notes/[Kubernetes 1.18] with CRI-O runtime. New features, changes, and known issues that pertain to
 {product-title} {product-version} are included in this topic.
 
 Red Hat did not publicly release {product-title} 4.5.0 as the GA version and,
-instead, is releasing {product-title} 4.5.1 as the GA version.
+instead, is releasing {product-title} 4.5.2 as the GA version.
```

Comment:
No! We GA'ed 4.5.1 (it was the version we first shipped for 4.5). There is no need for this. 
What version we announce with; is irrelevant, compared to what version we GA (actually ship with). 

cc: @reestr

Reply:
@sferich888 if we don't mention this, aren't we likely to get questions about why customers can't upgrade to 4.5.1 and instead are being upgraded to 4.5.2 directly?

Reply:
4.5.1 is the release for 4.5 GA.  We can't simply change that now without causing more confusion for customers. Some have already installed and getting support.  Let's please ensure to have consistent messaging. Only delay is with the marketing announcement and that is instead with 4.5.2.

Reply:
@reestr how can we have a different marketing announcement to the docs? Both docs and marketing need to have the same message?

If the marketing announcement says 4.5.2, and docs say 4.5.1, I can expect bugs coming, not to mention confusion in the field.

Reply:
Does something like this thread the needle?
`Red Hat did not publicly release {product-title} 4.5.0 as the GA version and, instead, released {product-title} 4.5.1 as the GA version. However, due to an upgrade issue, 4.5.1 has been pulled. A new upgrade path from 4.4.z to 4.5.1 and 4.5.1 to 4.5.2 will be available with the 4.5.2 release.`

Reply:
That seems OK to me.  Technically, we didn't pull anything, we just didn't publish a blog post.  I'd rather just say 4.5.1 was GA and change marketing language to say that 4.5 is GA and point to 4.5.2 as the most recent release.

Reply:
Updated with slight tweak:
```
Red Hat did not publicly release {product-title} 4.5.0 as the GA version and,
instead, released {product-title} 4.5.1 as the GA version. However, 4.5.1
has an upgrade issue (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856501[*BZ#1856501*]).
A new upgrade path from 4.4.z to 4.5.1 and 4.5.1 to 4.5.2 is available with the
4.5.2 release.
```